### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by kcimartinez**
Add responsive header component with sidebar and overlay for mobile view. Does not include working cart modal.

Live preview: https://audiophile-ecommerce-git-feature-header-muneyy.vercel.app?_vercel_share=bUVKOikQNeUt9qmkQmgdeI9JWK4CBwVz

### CODE REVIEW

File: src/components/Header/Sidebar.module.sass [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    height: 100vh
+    width: 200px
+    padding: 30px 20px
+    ul
+        display: flex
+        flex-direction: column
+        gap: 30px
+        list-style: none
+    transform-style: preserve-3d
+    // z-index: -1
```
Comment (eremocal) [inline src/components/Header/Sidebar.module.sass]: Please remove the commented code.

File: src/lib/contentful.js [hunk 2]
```diff
@@ -10,3 +10,12 @@ export const client = contentful.createClient({
 //   .getEntry('23ajyJGN6FZHErhMhAjYLu')
 //   .then((entry: any) => console.log(entry))
 //   .catch(console.error)
+
+// sample code for SSR
+// export async function getData() {
```
Comment (eremocal) [inline src/lib/contentful.js]: do you think we can remove the commented code?

File: src/components/Header/Sidebar.module.sass (hunk 1 above)
Comment (kcimartinez) [reply src/components/Header/Sidebar.module.sass]: removed commented code

File: src/lib/contentful.js (hunk 2 above)
Comment (kcimartinez) [reply src/lib/contentful.js]: removed commented code

**Review by eremocal — APPROVED** [review]

**Pull request merged**